### ISSUE TITLE
juju error

### ISSUE THREAD

**Issue by younes200** [scaleway/juju-scaleway#5]
Hi,
After configuring juju,  I'm getting this error with  `juju switch scaleway`

```
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/juju/juju/environs.ReadEnvironsBytes(0xc2080ec500, 0x5f, 0x25f, 0x5f, 0x0, 0x0)
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/juju/environs/config.go:252 +0x3fd
github.com/juju/juju/environs.ReadEnvirons(0x0, 0x0, 0x10, 0x0, 0x0)
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/juju/environs/config.go:288 +0x19d
main.(*SwitchCommand).Run(0xc208238b60, 0xc20810b5e0, 0x0, 0x0)
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/juju/cmd/juju/switch.go:76 +0x55
github.com/juju/cmd.(*SuperCommand).Run(0xc20823c000, 0xc20810b5e0, 0x0, 0x0)
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/cmd/supercommand.go:428 +0x630
github.com/juju/cmd.Main(0x2e1a830, 0xc20823c000, 0xc20810b5e0, 0xc20800a010, 0x2, 0x2, 0xc20802dbf0)
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/cmd/cmd.go:247 +0x30c
main.Main(0xc20800a000, 0x3, 0x3)
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/juju/cmd/juju/main.go:71 +0x507
main.main()
	/private/tmp/juju20150613-75732-3wtfvu/juju-core_1.24.0/src/github.com/juju/juju/cmd/juju/main.go:233 +0x42

goroutine 5 [syscall]:
os/signal.loop()
	/usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/usr/local/Cellar/go/1.4.2/libexec/src/os/signal/signal_unix.go:27 +0x35
```

Any idea ?
Thanks

**Comment by edouardb** [comment]
Can you paste your environment?

**Comment by edouardb** [comment]
Does it look like this?

```
environments:
    scaleway:
    type: manual
    bootstrap-host: null
    bootstrap-user: root
```

**Comment by younes200** [comment]
yes, as mentioned in README file

**Comment by edouardb** [comment]
Environment is stored at  `~/.juju/environments.yaml` right?

Can you give me your juju version?

**Comment by younes200** [comment]
$ juju --version
1.24.0-yosemite-amd64

**Comment by edouardb** [comment]
Can you update to 1.24.3 and give a new try?

**Comment by kdeldycke** [comment]
Looks like the underlaying issue was fixed by #6 .

I'm then closing the issue unless there is still a problem with the upcoming 1.0.3.
